tui/profiles.sh

```diff
@@ -6,6 +6,9 @@ source "tui/locales/$LOCALE/profiles.sh"
 active_profile="ovos"
 available_profiles=(ovos satellite listener)
 
+# adding back button
+available_profiles+=($BACK_BUTTON)
```

Comment:
Same as above, `$BACK_BUTTON` should not be in this array and I think it should not be in any array at all.